client/src/components/Helper/SasCodeHelper.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```

Comment:
I wonder if this belongs in `client/src/components/utils.ts`. NOTE: It looks like there may be some conflicts with this pull request and https://github.com/sassoftware/vscode-sas-extension/pull/473. 

We'll need to make sure we take those changes into account, depending on who merges first.

Reply:
The utils.ts has functions for different purpose I don't think it is good, I prefer to organize functions which have similar purpose together. such as SASCodeHelper for functions to operate sas code, SettingHelper for functions to manipulate settings.

Reply:
That makes sense. I mentioned splitting the utils into distinct files in #473 . I think what you have for now is probably fine, but I'm going to assign myself a future task of reorganizing this a bit after your code and Casey's are merged. I'm thinking the final structure will be something like...

```
|- client/src/components/utils/
     |- sasCodeHelper.ts (may or may not include Casey's code)
     |- settingHelper.ts
     |- throttle.ts (will contain what is currently in `utils.ts`)
```